environments/prod/courttribunalfinder-service-gov-uk.yml

```diff
@@ -0,0 +1,20 @@
+ns: []
```

Comment:
you also need to add the zone in the prod directory, e.g. like:

https://github.com/hmcts/azure-public-dns/blob/master/components/prod/apply-divorce.tf

make sure to update all the resource names to your zone

Reply:
added in